legend-engine-server/pom.xml

```diff
@@ -162,6 +162,16 @@
             <artifactId>legend-engine-xt-persistence-cloud-grammar</artifactId>
             <scope>runtime</scope>
         </dependency>
+        <dependency>
```

Comment:
I don't see any new code being added to the server module. Is this dependency required ?

Reply:
It is not, I have removed it.